src/main/java/de/neemann/digital/gui/components/graphics/GraphicCard.java

```diff
@@ -158,12 +158,15 @@ public DataField getMemory() {
     private void updateGraphic(boolean bank) {
         if (paintPending.compareAndSet(false, true)) {
             SwingUtilities.invokeLater(() -> {
-                if (graphicDialog == null || !graphicDialog.isVisible()) {
+                if ((graphicDialog == null || !graphicDialog.isVisible()) && getModel().runningInMainFrame()) {
```

Comment:
Wouldn't it be better to add an "if (getModel().runningInMainFrame()) {}" on the very outside? If the model is running in the console and no graphics is present, no swing events need to be generated.